### DISCUSSION TITLE
Not building ..

### DISCUSSION THREAD

**Discussion by sorydima** [codemagic-ci-cd/codemagic-docs#2558] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Run the Flutter Build project:

https://github.com/sorydima/Marina.Moda-.git

On the CodeMagic!

### Expected results

Build done! ..

### Actual results

![Снимок экрана от 2024-05-27 14-53-45](https://github.com/codemagic-ci-cd/codemagic-docs/assets/43252008/4f951f42-d441-4f7c-aa64-4a9ffb3ed8f8)

![Снимок экрана от 2024-05-27 14-53-53](https://github.com/codemagic-ci-cd/codemagic-docs/assets/43252008/86ac4a0d-2677-440f-af15-a24646425f5c)

                                       ^
lib/widgets/no_internet_widget.dart:78:21: Error: No named parameter with the name 'primary'.
                    primary: Theme.of(context).primaryColor,
                    ^^^^^^^
../programs/flutter_3_22_0/packages/flutter/lib/src/material/elevated_button.dart:196:22: Context: Found this candidate, but the arguments don't match.
  static ButtonStyle styleFrom({
                     ^^^^^^^^^
lib/widgets/not_found.dart:84:21: Error: No named parameter with the name 'primary'.
                    primary: Theme.of(context).primaryColor,
                    ^^^^^^^
../programs/flutter_3_22_0/packages/flutter/lib/src/material/elevated_button.dart:196:22: Context: Found this candidate, but the arguments don't match.
  static ButtonStyle styleFrom({
                     ^^^^^^^^^
lib/widgets/not_found.dart:94:59: Error: The argument type 'Uri' can't be assigned to the parameter type 'WebUri?'.
 - 'Uri' is from 'dart:core'.
 - 'WebUri' is from 'package:flutter_inappwebview_platform_interface/src/web_uri.dart' ('../.pub-cache/hosted/pub.dev/flutter_inappwebview_platform_interface-1.0.10/lib/src/web_uri.dart').
                          urlRequest: URLRequest(url: Uri.parse(widget.url)));
                                                          ^
lib/widgets/no_internet.dart:15:16: Error: A value of type 'List<ConnectivityResult>' can't be assigned to a variable of type 'ConnectivityResult'.
 - 'List' is from 'dart:core'.
 - 'ConnectivityResult' is from 'package:connectivity_plus_platform_interface/src/enums.dart' ('../.pub-cache/hosted/pub.dev/connectivity_plus_platform_interface-2.0.0/lib/src/enums.dart').
      result = await _connectivity.checkConnectivity();
               ^
Target kernel_snapshot_program failed: Exception

FAILURE: Build failed with an exception.

* Where:
Script '/Users/builder/programs/flutter_3_22_0/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy' line: 1675

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command '/Users/builder/programs/flutter_3_22_0/bin/flutter'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 57s
Running Gradle task 'bundleRelease'...                            118.6s
Gradle task bundleRelease failed with exit code 1

Build failed :|
Failed to build for Android

Build overview
ID:
66545ba5991f16a225c2f793
Please provide this ID when you contact our support team
Index:
5
Status:
failed
Workflow:
Default Workflow
Started by:
Dmitry Sorokin (@sorydima)
Dmitry Sorokin (@sorydima)sorydima@gmail.com
Started:
2 hours ago
May 27th, 2024 at 13:08 MSK
Duration:
3m 56s
Machine:
Mac mini M1
Branch:
main
Commit:
57c80b2

Current configuration
Flutter channel:
master
Mode:
release
Xcode version:
edge
Build for:
Android & iOS

### Build id (optional)

66545ba5991f16a225c2f793

**Comment by himesh-cm** [comment]
The thread is continued on the intercom.

**Comment by himesh-cm** [comment, marked as answer]
The thread was resolved on the intercom
